articles/manual/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Q# Manual | Microsoft Docs 
+description: Q# Manual
+author: E. Gonzalez
+ms.author: v-edsanc@microsoft.com 
+ms.date: 07/31/2019
+ms.topic: article
+uid: microsoft.quantum.manual.intro
+---
+
+# Q# Manual
+
+## What is Q#?
+
+Q# (Q-sharp) is a domain-specific programming language used for
+expressing quantum algorithms.
+It is to be used for writing subroutines that execute on an adjunct
+quantum processor, under the control of a classical host program and computer.
+Until quantum processors are widely available, Q# subroutines execute on a simulator.
+
+A natural model for quantum computation is to treat the quantum computer
+as a coprocessor, similar to that used for GPUs, FPGAs, and other adjunct
+processors.
+The primary control logic runs classical code on a classical "host" computer.
+When appropriate and necessary, the host program can invoke a subroutine
+that runs on the adjunct processor.
+When the subroutine completes, the host program gets access to the
+subroutine's results.
+
+Q# provides a small set of primitive types, along with two ways
+(arrays and tuples) for creating new, structured types.
+It supports a basic procedural model for writing programs,
```

Comment:
I think this sentence made a bit more sense in the original version than it does here; we might not want to open by implying that Q# only supports a basic model, as that works against teaching the reader about the high-level functionality offered by Q#.

Reply:
I don't fully get the idea you have in mind here. Check 0d24276bcc258abdaa682dd973a725279954fb4b to see if now it's better.